### PR TITLE
Filterable Image Gallery

### PR DESCRIPTION
**Pull request by PritomDutta89**
## Related Issue
 - Filterable Image Gallery

Closes: #1492 
### Describe the changes you've made
Added a project where you can filter images by your choice, made with HTML, CSS, JS.

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots

IMAGE FILTER 1

IMAGE FILTER 2

IMAGE FILTER 3

IMAGE FILTER 4

IMAGE FILTER 5

IMAGE FILTER 6

IMAGE FILTER 7

### CODE REVIEW

**Comment by PritomDutta89** [comment]
Please @ridsuteri @991rajat @urvashi-code1255 review this PR.

**Review by ridsuteri — CHANGES_REQUESTED** [review]
@PritomDutta89 the image of the project is not showing in the project card, please fix it, also resolve merge conflicts :)

**Review by PritomDutta89 — COMMENTED** [review]
now check

**Review by ridsuteri — CHANGES_REQUESTED** [review]
sorry @PritomDutta89 , the issue still persists :(

**Comment by PritomDutta89** [comment]
@ridsuteri what is project card?

**Comment by ridsuteri** [comment]
> @ridsuteri what is project card?
this
Screenshot (81)_LI

**Comment by PritomDutta89** [comment]
@ridsuteri Please can you help me how to fix it

**Review by PritomDutta89 — COMMENTED** [review]
by mistake

**Comment by PritomDutta89** [comment]
@ridsuteri Done, now check

File: script.js [hunk 1]
```diff
@@ -1418,6 +1419,14 @@ let projectData = [
...
      projectImage: 'assets/Images/CaptchaValidator.png',
      projectUrl: 'public/CaptchaValidator.html'
+   },
+   {
+    projectName: "Filterable Image Gallery",
+    projectImage: "assets/Images/Filterable_Image_Gallery/Filterable_Image_Gallery.png",
+    projectUrl: "public/Filterable_Image_Gallery.html",
+  }
+  
+]
```
Comment (991rajat) [inline script.js:1429]: This is ending the list which is introducing error. Remove the square bracket.
But there are a lot of projects down below.

File: script.js [hunk 2]
```diff
@@ -78,6 +78,7 @@ let projectData = [
   {
     projectName: "RESTAURANT WEBSITE",
     projectImage: "assets/GIFs/Restaurant.gif",
+    projectUrl: "public/Restaurant.html",
     projectUrl: "public/Restaurant.html"
```
Comment (991rajat) [inline script.js:82]: Remove this line. It's duplicate.

**Review by 991rajat — COMMENTED** [review]
Look at the changes mentioned.

File: script.js (hunk 2 above)
Comment (PritomDutta89) [reply script.js]: done

**Comment by PritomDutta89** [comment]
now check

**Comment by 991rajat** [comment]
Remove these. They already exist.
image

**Comment by PritomDutta89** [comment]
I don't understand, I have to these things here, I didn't add these things, I just added my file only

**Comment by 991rajat** [comment]
Looks Good.

**Review by 991rajat — APPROVED** [review]
Good Work

**Comment by 991rajat** [comment]
Congratulations on merging of first PR. :tada: 
Thanks for this awesome contribution.

**Pull request merged**